templates/error.html

```diff
@@ -0,0 +1,139 @@
+{% extends 'base.htm' %}
+
+{% block content %}
+    <!-- This template was made by Colorlib (https://colorlib.com)
+         Please visit our website for more awesome templates, themes and tools. -->
+	<div id="notfound">
```

Comment:
Since this is the base error template, let's choose more generic `id`s (to describe error elements rather than specifically 404 elements).

Reply:
Note: The CSS rules will need to be updated when the ids and classes are changed.

Reply:
I can update that. That was just what came with the Colorlib template.

Reply:
Are `HTTP-error` and `HTTP-error-content` acceptable?